.github/workflows/head-dependencies.yml

```diff
@@ -0,0 +1,147 @@
+name: HEAD of dependencies
+
+on:
+  # Run daily at 1:23 UTC
+  schedule:
+  - cron:  '23 1 * * *'
+  workflow_dispatch:
+
+jobs:
+  release-candidates:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+        python-version: ['3.10']
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v3
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install core fonts
+      if: runner.os == 'Linux'
+      run: |
+        echo ttf-mscorefonts-installer msttcorefonts/accepted-mscorefonts-eula select true | sudo debconf-set-selections
+        sudo apt-get install ttf-mscorefonts-installer
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip setuptools wheel
+        python -m pip --quiet install --upgrade --pre .[test]
+
+    - name: List installed Python packages
+      run: python -m pip list
+
+    - name: Test with pytest
+      run: |
+        pytest -r sa --mpl --mpl-results-path=pytest_results
+
+  matplotlib:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python-version: ['3.10']
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v3
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install core fonts
+      run: |
+        echo ttf-mscorefonts-installer msttcorefonts/accepted-mscorefonts-eula select true | sudo debconf-set-selections
+        sudo apt-get install ttf-mscorefonts-installer
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip setuptools wheel
+        python -m pip --no-cache-dir --quiet install --upgrade .[test]
+        python -m pip install --pre --upgrade --extra-index-url https://pypi.anaconda.org/scipy-wheels-nightly/simple matplotlib
+
+    - name: List installed Python packages
+      run: python -m pip list
+
+    - name: Test with pytest
+      run: |
+        pytest -r sa --mpl --mpl-results-path=pytest_results
```

Comment:
When I checked this with a test run `test_histplot_kwargs` is failing against the nightly build wheel. Something to keep on the radar https://github.com/scikit-hep/mplhep/runs/5790229209?check_suite_focus=true

Reply:
Done! :)

Reply:
Oh wait. When I ran on a debug branch I got

```
Warning: No files were found with the provided path: pytest_results. No artifacts will be uploaded.
```

let me look into that.

Reply:
Issue #375.